src/firmware/boards/robot_stm32h7/io/chicker.c

```diff
@@ -0,0 +1,35 @@
+#include "firmware/boards/robot_stm32h7/io/chicker.h"
+
+#include "firmware/app/logger/logger.h"
+
+void io_chicker_kick(float speed_m_per_s)
+{
+    TLOG_DEBUG("io_chicker_kick: %f", speed_m_per_s);
```

Comment:
Why do we have debug print statements here but not in the below functions? Seems this would get spammed a lot in some cases.

Reply:
ah yeah we don't need this, this will be replaced as soon as mainboard rev 2.1 gets here, mainboard rev 2.0 has issues with the power board -> mainboard connector